### DISCUSSION TITLE
HP EliteBook 8560p

### DISCUSSION THREAD

**Discussion by jessylou** [chris1111/HP-Probook-EliteBook-Package-Creator#69] [General]
Hi,

I have an old HP EliteBook 8560p. I fear as it's 6 series all your work won't be usable on this old machine.

Thanks in advance for your response (and tips if you have some).

Jessy

**Comment by chris1111** [comment]
Follow [6 seires](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator)

**Comment by jessylou** [comment]
Many thanks Chris,

I already saw the work you've done based on Clover. Since I moved all my systems on OC, I would love to try it.

So far, I detected one issue running you Install Media HP Laptop OC. As I just figured it out, let me share with you.

When the machine has one "macOS" system mounted, the final step just crashes. I renamed the volume: everything went just fine.

My test has been done based on the latest OC package (august 21st): https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator-OC

Based on the detailed video you've done (https://www.youtube.com/watch?v=gaxkglymYkE), the crash happens at 10:36 / 14:39. Hoe this will help.

To finish, OpenCore does not support iGPU on macOS 10.14+, therefore I just give up.

Anyway: THANK YOU for all the work you do and the support you provide to many users!

**Comment by chris1111** [comment]
`9: 54 m `
you have to select the destination USB media

![Screenshot 2023-08-25 at 06 27 56](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator/assets/6248794/03435af2-2f0c-47cb-84a8-574fd2d2c185)

If the installer crash at this point at `10:36 / 14:39 `you can provided the Full Log

![Screenshot 2023-08-25 at 06 31 58](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator/assets/6248794/4b7eb744-1e33-4813-8c95-96f3b2760d29)

![Screenshot 2023-08-25 at 06 32 41](https://github.com/chris1111/HP-Probook-EliteBook-Package-Creator/assets/6248794/da42df9c-082a-4ff9-9a59-c127c248402d)

Maybe a USB issue ? There are USB sticks works really badly in macOS

**Reply by jessylou** [reply]
Many thanks Chris for your constant effort to provide with the best solution.

1). Thanks to other information got from your site and videos, I found out how to get logs... Unfortunately it has been discovered after I renamed the faulty (my understanding) partition.

2). For sure this is not coming from the USB 3.0 stick I used on each installation/upgrade for years. Very efficient, very strong, not that fast but reliable.

Thank you again.

Wishing you a nice day!
